apbs/geometry/sphere.py

```diff
@@ -13,7 +13,7 @@ def spherical_distribution(self, npoints: int) -> Surface:
 
         Shamelessly copied over from the vacc routine.
```

Comment:
We should consider replacing this with https://github.com/Electrostatics/osmolytes/blob/f8ac99827ba199b359ccf141b604a2b7362f9dbf/osmolytes/sasa.py#L31

Reply:
Would we just add the osmolytes package and remove many of the geometry related functions?